crates/fgumi-bam-io/CHANGELOG.md

```
@@ -4,6 +4,23 @@ All notable changes to this project will be documented in this file.
+
+### Performance
+
+- Cache next milestone threshold for log_if_needed fast path ([#345](https://github.com/fulcrumgenomics/fgumi/pull/345))
+
+### Refactor
+
+- Extract fgumi-bam-io and fgumi-sort workspace crates ([#323](https://github.com/fulcrumgenomics/fgumi/pull/323))
+
+<!-- generated by git-cliff -->
+
 ### Refactor
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)